src/Core_Fix_Heterochromia/Core.HeterochromiaFix.Hooks.cs

```diff
@@ -17,7 +17,7 @@ internal static class Hooks
             internal static void LoadFileLimitedPostfix(ChaFileControl __instance)
             {
                 //Find the toggle set
-                var cvsEye02 = CustomBase.Instance.gameObject.GetComponentInChildren<CvsEye02>(true);
+                var cvsEye02 = CustomBase.Instance?.gameObject.GetComponentInChildren<CvsEye02>(true);
```

Comment:
This might still throw a nullref after exiting maker twice, since the Instrance can be a destroyed Unity Object that is not detected as null by the ? operator. Using a `!= null` avoids the issue.
```suggestion
                var cvsEye02 = CustomBase.Instance != null ? CustomBase.Instance.gameObject.GetComponentInChildren<CvsEye02>(true) : null;
```

Reply:
Thanks for the review. Fixed.
This spec is a Unity trap.